lib/stringifier.js

```diff
@@ -124,8 +124,14 @@ const stringifySvg = (data, userOptions = {}) => {
     config.textEnd += eol;
   }
   let svg = stringifyNode(data, config, state);
-  if (config.finalNewline && svg.length > 0 && svg[svg.length - 1] !== '\n') {
+  if (config.finalNewline && svg.length > 0 && svg[svg.length - 1] !== eol) {
```

Comment:
This will never work with crlf as there are 2 characters.

Reply:
You're right, I forgot about that. Will adjust it.

Reply:
Should be fixed by 95b0122caef622a83f24a71af3a2c3018bffc7cc.